docs/concepts/deployments-ux.md

```diff
@@ -1,5 +1,5 @@
 ---
-description: Learn how Prefect projects allow you to easily manage your code and deployments.
+description: Learn how to easily manage your code and deployments.
```

Comment:
We probably want to change the title of this page to something like 'Managing Deployments'.

Reply:
Sounds good, I've updated the concepts page to not include references to projects